retrace/glws_egl_xlib.cpp

```diff
@@ -218,6 +225,12 @@ class EglDrawable : public Drawable
         eglSwapBuffers(eglDisplay, surface);
         processKeys(window);
     }
+
+    void swapBuffersWithDamage(int *rects, int nrects) override {
+        bindAPI(api);
```

Comment:
Likewise, here one should check if the EGL extension is supported, and do nothing otherwise.